### PR TITLE
fix: only allow ITAdmin to list projects by ETC

### PR DESCRIPTION
**Pull request by Bingjiling**
Issue #, if available:

Description of changes:
* only allow ITAdmin to list projects by ETC

Checklist:

* [x] Have you successfully deployed to an AWS account with your changes?
* [x] Have you written new tests for your core changes, as applicable?

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: solutions/swb-reference/integration-tests/tests/isolated/environmentTypeConfigs/listProjectsByEtc.test.ts [hunk 1]
```diff
@@ -0,0 +1,113 @@
+/*
```
Comment (kpark277) [inline solutions/swb-reference/integration-tests/tests/isolated/environmentTypeConfigs/listProjectsByEtc.test.ts:1]: Nit: tests exist under `integration-tests/tests/isolated/projectEnvTypeConfigs/listProjects.test.ts`. Look to consolidate these.

File: common/changes/@aws/swb-reference/yanyu-list-poject-by-etc_2023-06-13-19-21.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (kpark277) [inline common/changes/@aws/swb-reference/yanyu-list-poject-by-etc_2023-06-13-19-21.json:1]: Remove

File: solutions/swb-reference/integration-tests/tests/isolated/environmentTypeConfigs/listProjectsByEtc.test.ts [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+    paSession = paabResources.pa1Session;
+    researcherSession = paabResources.rs1Session;
+  });
+
+  afterAll(async () => {
+    await paabHelper.cleanup();
+    await setup.cleanup();
+  });
+
+  describe('ITAdmin tests', () => {
```
Comment (nguyen102) [inline solutions/swb-reference/integration-tests/tests/isolated/environmentTypeConfigs/listProjectsByEtc.test.ts:36]: Should these tests go into this preexisting file?
https://github.com/aws-solutions/research-service-workbench-on-aws/blob/2645d12a07ae9b9ba185e234713c55639e96291d/solutions/swb-reference/integration-tests/tests/isolated/projectEnvTypeConfigs/listProjects.test.ts

File: solutions/swb-reference/integration-tests/tests/isolated/environmentTypeConfigs/listProjectsByEtc.test.ts (hunk 1 above)
Comment (Bingjiling) [reply solutions/swb-reference/integration-tests/tests/isolated/environmentTypeConfigs/listProjectsByEtc.test.ts:1]: Ah I did not know there's an existing one. Will consolidate.

File: solutions/swb-reference/src/postDeployment/authorizationSetup.ts [hunk 4]
```diff
@@ -48,6 +48,10 @@ export default class AuthorizationSetup {
     ]);
     const environmentTypePermissions = this._mapActions(itAdmin, SwbAuthZSubject.SWB_ENVIRONMENT_TYPE);
     const environmentTypeConfigPermissions = this._mapActions(itAdmin, SwbAuthZSubject.SWB_ETC);
+    const environmentTypeConfigListPermissions = this._mapActions(
+      itAdmin,
+      SwbAuthZSubject.SWB_PROJECT_LIST_BY_ETC
```
Comment (nguyen102) [inline solutions/swb-reference/src/postDeployment/authorizationSetup.ts:53]: Should we passed in `['READ']` for the list of actions?

**Review by nguyen102 — COMMENTED** [review]
lgtm

**Review by kpark277 — APPROVED** [review]

**Pull request merged**